tools/add_pr_link_to_jira.py

```diff
@@ -60,7 +52,16 @@ def dummyApplicationLinks():
         isVerbose = True
         logging.getLogger("__main__").setLevel(logging.DEBUG)
 
-    j = jira_netrc_login(args.netrc)
+    if args.jira_user_password is None:
+        username = password = None
+    else:
+        username, password = args.jira_user_password.split(":", 1)
```

Comment:
Or add function to `utils` module that gets the `username` and `password` instead of duplicating that part every time.
e.g.

```python
def get_login_credentials(args, field):
    if not hasattr(args, field) or getattr(args, field) is None:
        return None, None

    return args.jira_user_password.split(":", 1)
```

and then just call it 
```python
username, password = get_login_credentials(args, "jira_user_password")

or 

username, password = get_login_credentials(args, "user_password")

```

Reply:
sounds good
implemented one function to take care of it